controller/role.go

```diff
@@ -156,11 +156,22 @@ func (role *Role) reconcileRole(instances map[string]*Instance, secrets map[stri
 	}
 
 	roleRepository := repository.NewRoleRepository(conn)
-	err = roleRepository.Create(role.Spec.RoleName)
+
+	var exists bool
+	exists, err = roleRepository.DoesRoleExist(role.Spec.RoleName)
 	if err != nil {
 		return err
 	}
 
+	if exists {
+		log.Infof("role %s already exists, skipping creation", role.Spec.RoleName)
```

Comment:
same here, the current logging schema is for example : "could not find instance '%s' in namespace '%s' for role '%s' in namespace '%s'",